packages/pn-personafisica-webapp/src/components/Contacts/SercqSendContactItem.tsx

```diff
@@ -298,17 +303,27 @@ const SercqSendContactItem: React.FC = () => {
   return (
     <DigitalContactsCard
       title={
-        value ? t('legal-contacts.sercq-send-title', { ns: 'recapiti' }) : <SercqSendCardTitle />
+        value ? (
+          t('legal-contacts.sercq-send-title', { ns: 'recapiti' })
+        ) : (
+          <SercqSendCardTitle pecEnabled={!!defaultPECAddress?.pecValid} />
+        )
+      }
+      subtitle={
+        defaultPECAddress?.pecValid
+          ? t('legal-contacts.sercq-send-description-pec-enabled', { ns: 'recapiti' })
+          : t('legal-contacts.sercq-send-description', { ns: 'recapiti' })
       }
-      subtitle={t('legal-contacts.sercq-send-description', { ns: 'recapiti' })}
       expanded
       sx={{
-        borderBottomLeftRadius: value ? 0 : 4,
```

Comment:
Why did you remove this?

Reply:
It should be wrong. Because the borderBottom props was used to rendere the way in the case fo sercq enabled